internals/search_fulltext.py

```diff
@@ -168,21 +182,24 @@ def canonicalize_string(s: str) -> str:
   return ' ' + canonicalized + ' '  # Avoids matching partial words.
 
 
-def post_process_phrase(phrase: str, feature_ids: list[int]) -> list[int]:
+def post_process_phrase(
+    phrase: str, feature_ids: list[int],
+    field_name: str|None = None) -> list[int]:
   """Fetch the given features and check if they really have the phrase."""
   features = get_future_results(get_entries_by_id_async(feature_ids))
   canon_phrase = canonicalize_string(phrase)
   result = []
   for fe in features:
-    feature_strings = get_strings(fe)
+    feature_strings = get_strings(fe, field_name=field_name)
     has_phrase = any(canon_phrase in canonicalize_string(fs)
                      for fs in feature_strings)
     if has_phrase:
       result.append(fe.key.integer_id())
   return result
 
 
-def search_fulltext(textterm: str) -> Optional[list[int]]:
+def search_fulltext(
+    textterm: str, field_name: str|None = None) -> Optional[list[int]]:
   """Return IDs of features that have some word(s) from phrase."""
```

Comment:
Update the comment to incorporate field_name

Reply:
Done.